### PR TITLE
Document the enhancements to ILM Health Indicator

### PR DESCRIPTION
**Pull request by HiDAl**

### CODE REVIEW

File: docs/reference/health/health.asciidoc [hunk 1]
```diff
@@ -365,6 +365,43 @@ watermark threshold>>.
 `policies`::
     (int) The number of index lifecycle policies that the system is managing.
 
+`stagnating_indices`::
+    (int) the number of indices –managed by an Index Lifecycle Policy– that has been stagnant longer than expected.
```
Comment (gmarouli) [inline docs/reference/health/health.asciidoc:369]: I would recommend to use the macros, either `{ilm}` or `{ilm-init}` if it has been mentioned before, and rephrase it to `managed by ILM`

File: docs/reference/settings/health-diagnostic-settings.asciidoc [hunk 2]
```diff
@@ -35,3 +35,13 @@ cluster wide health aspects such as disk space.
 `health.reporting.local.monitor.interval`::
 (<<cluster-update-settings,Dynamic>>) Determines the interval in which each node of the cluster monitors aspects that
 comprise its local health such as its disk usage.
+
+`health.ilm.max_time_on_action`::
+(<<cluster-update-settings,Dynamic>>) The minimum amount of time an index has to be in an ILM action before it is considered stagnant. Defaults to `1d` (1 day).
```
Comment (gmarouli) [inline docs/reference/settings/health-diagnostic-settings.asciidoc:40]: Same here about ILM, better to use the macro `{ilm-init}`.

File: docs/reference/settings/health-diagnostic-settings.asciidoc [hunk 3]
```diff
@@ -35,3 +35,13 @@ cluster wide health aspects such as disk space.
...
+
+`health.ilm.max_time_on_step`::
+(<<cluster-update-settings,Dynamic>>) The minimum amount of time an index has to be in an ILM step before it is considered stagnant. Defaults to `1d` (1 day).
```
Comment (gmarouli) [inline docs/reference/settings/health-diagnostic-settings.asciidoc:43]:
```suggestion
(<<cluster-update-settings,Dynamic>>) The minimum amount of time an index has to be in an {ilm-init} step before it is considered stagnant. Defaults to `1d` (1 day).
```

File: docs/reference/settings/health-diagnostic-settings.asciidoc [hunk 4]
```diff
@@ -35,3 +35,13 @@ cluster wide health aspects such as disk space.
...
+
+`health.ilm.max_retries_per_step`::
+(<<cluster-update-settings,Dynamic>>) The minimum amount of times an index has retried by an ILM step before it is considered stagnant. Defaults to `100`
```
Comment (gmarouli) [inline docs/reference/settings/health-diagnostic-settings.asciidoc:46]:
```suggestion
(<<cluster-update-settings,Dynamic>>) The minimum amount of times an index has retried by an {ilm-init} step before it is considered stagnant. Defaults to `100`
```

File: docs/reference/settings/health-diagnostic-settings.asciidoc (hunk 2 above)
Comment (HiDAl) [reply docs/reference/settings/health-diagnostic-settings.asciidoc:40]: didn't know about those macros. Thanks Mary

File: docs/reference/settings/health-diagnostic-settings.asciidoc [hunk 5]
```diff
@@ -35,3 +35,13 @@ cluster wide health aspects such as disk space.
 `health.reporting.local.monitor.interval`::
 (<<cluster-update-settings,Dynamic>>) Determines the interval in which each node of the cluster monitors aspects that
 comprise its local health such as its disk usage.
+
+`health.ilm.max_time_on_action`::
+(<<cluster-update-settings,Dynamic>>) The minimum amount of time an index has to be in an {ilm-init} action before it is considered stagnant. Defaults to `1d` (1 day).
```
Comment (gmarouli) [inline docs/reference/settings/health-diagnostic-settings.asciidoc:40]: It's good practice the first time we use an acronym in a page to spell it out, this will ensure that in every page the reader can find the reference and they don't have to start looking around. For example: adding `{ilm} ({ilm-init})` here is enough.

**Review by gmarouli — APPROVED** [review]
LGTM, one small adjustment and you are good to go. 🚀
